code/gui/mainwindow.ui

```diff
@@ -34,7 +34,14 @@
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
-        <widget class="QLineEdit" name="move_command_edit"/>
+        <widget class="QLineEdit" name="move_command_edit">

```

Comment:
Nope, we don't have that, *yet*. It would be a very low-priority enhancement, so possibly not in the plan for the near future.